From fdf784e5090524d9af6e8f02161c05a6d4592774 Mon Sep 17 00:00:00 2001
From: mydongistiny <jaysonedson@gmail.com>
Date: Wed, 4 Nov 2015 11:04:11 -0800
Subject: [PATCH] oneplus2: Remove verify

 init.oneplus2.rc | 8 --------
 1 files changed, 0 insertions(+), 8 deletions(-)
diff --git a/init.oneplus2.rc b/init.oneplus2.rc
index 4fb4c10..af49acd 100755
--- a/init.oneplus2.rc
+++ b/init.oneplus2.rc
@@ -23,8 +23,6 @@ on early-init
     chown system system /sys/kernel/debug/kgsl/proc
 
 on init
-    # Load persistent dm-verity state
-    verity_load_state
 
     # Set permissions for persist partition
     mkdir /persist 0771 system system
@@ -60,12 +58,6 @@ on early-boot
     # set RLIMIT_MEMLOCK to 64MB
     setrlimit 8 67108864 67108864
 
-    # Adjust parameters for dm-verity device
-    write /sys/block/dm-0/queue/read_ahead_kb 2048
-
-    # Update dm-verity state and set partition.*.verified properties
-    verity_update_state
-
 on boot
     # execute script to set initial CPU settings
     # don't run as a service to avoid race conditions
